### PR TITLE
Update the SDK

### PR DESCRIPTION
**Pull request by simon816**
Update to ForgeGradle 2
Update reference to SpongeForge instead of Sponge
Provide simple plugin template rather than a more complete plugin

### CODE REVIEW

File: sdk/build.gradle [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
 repositories {
+    mavenCentral()
+    maven {
+        name = "sponge"
+        url = "http://repo.spongepowered.org/maven"
+    }
     maven {
-        name "sponge"
-        url "http://repo.spongepowered.org/maven/"
+        name "spongeforge"
```
Comment (kashike) [inline sdk/build.gradle]: `name "forge"`

Comment (Minecrell) [inline sdk/build.gradle]: Also use `=` for consistency with above

File: sdk/build.gradle [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
             url = "http://files.minecraftforge.net/maven"
         }
-        maven {
-            name = "sonatype"
-            url = "https://oss.sonatype.org/content/repositories/snapshots/"
-        }
     }
     dependencies {
-        classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
+        classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-20160116.192356-17'
```
Comment (liach) [inline sdk/build.gradle]: Why not just `2.1-SNAPSHOT`?

Comment (simon816) [inline sdk/build.gradle]: I used the exact version SpongeForge was on at the time

**Comment by Minecrell** [comment]
I'm going to close this, once I find the time I'll start on getting something into SpongeGradle which will allow for easier testing with both implementations.

**Pull request closed without merging**